README.md

```diff
@@ -60,20 +60,22 @@ node build/main.js
 **Endpoints**
 
 - **POST /get_pot**: Accepts a `visitor_data` (unauthenticated), `data_sync_id` (authenticated) or an empty body in the request body. If no identifier is passed, a new unauthenticated `visitor_data` will be generated. Returns `po_token` and the associated identifier `visit_identifier`.
-- **POST /invalidate_caches**: Resets the PO token cache, forcing new tokens to be generated on next fetch
+- **POST /invalidate_caches**: Resets the PO token cache, forcing new tokens to be generated on next fetch.
 </details>
 
 #### (b) Generation Script Option
 
-The generation script needs to be transpiled to Javascript before it can be used by the plugin.
+1. Transpile the generation script to Javascript:
 
 ```shell
 cd server/
 yarn install --frozen-lockfile
 npx tsc
 ```
 
-Make sure `node` is available in your `PATH`.
+2. Move the `server` directory with transpiled Javascript to `~/bgutil-ytdlp-pot-provider/` if you're using Linux or MacOS.
```

Comment:
Would it be easier to understand if we just said to download the source code to their home directory? Instead of asking them to move a specific directory into a new folder?

Reply:
maybe add a `clone the repo/download the source code` step